app/src/main/java/com/fstyle/structure_android/MainActivity.kt

```diff
@@ -0,0 +1,15 @@
+package com.fstyle.structure_android
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * Create by DaoLQ
+ */
+class MainActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/04a1b709612ae5ecce8bed53309e05fa09d012b8/app/src/main/java/com/fstyle/structure_android/MainActivity.kt#L13"></span>Unexpected indentation (4) (it should be 6) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>